src/collect-adb.rst

```diff
@@ -1,108 +1,205 @@
 
-  $ adb shell rm -d /sdcard/odk/forms/my_form.xml
+  $ adb shell rm -d /sdcard/odk/forms/form.xml
 
 .. _downloading-forms:
 
 Downloading forms to your computer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To download a completed form or form instance from the computer, run:
+To download a completed form or form instance from the computer:
 
 .. code-block:: console
 
-  $ adb pull /sdcard/odk/forms/my_form.xml
+  $ adb pull /sdcard/odk/forms/form.xml
 
+  
+.. _adb-dev-tasks:
+  
+Developer tasks and troubleshooting with ADB
+-----------------------------------------------
+  
 .. _downloading-database-with-adb:
 
-Downloading database
-~~~~~~~~~~~~~~~~~~~~~~
+Downloading Collect databases
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Collect stores settings and form state information
+in several SQLite databases, 
+which you can pull onto your local computer.
 
-Developers might also need to check the entries in the database from the computer. In such case pull the database file from the SD card and use any **SQLite visualizer** to view it. To pull the database into the computer, run:
+To see a list of available databases:
 
 .. code-block:: console
   
-  $  adb -pull /sdcard/odk/database/database.name
+  $  adb ls /sdcard/odk/database/
 
-.. _saving-screenshot-with-adb:
+To pull a database locally:
 
-Saving screenshot
-~~~~~~~~~~~~~~~~~~
+.. code-block:: console
+  
+  $  adb pull /sdcard/odk/database/{database-name}
 
-For taking a screenshot, run:
+.. _saving-screenshot-with-adb:
+
+Taking screenshots
+~~~~~~~~~~~~~~~~~~~~~
 
 .. code-block:: console
 
-  $ adb exec-out screencap /sdcard/screen.png
+  $ adb exec-out screencap /sdcard/image-name.png
 
-Here, the image will be stored as ``screen.png`` which can be downloaded to the computer by running:
+To pull the saved image locally:
 
 .. code-block:: console
 
-  $ adb pull /sdcard/screen.png
+  $ adb pull /sdcard/image-name.png
 
 .. note::
 
-  You can also use ODK docs program to get a screenshot by referring to the instructions given in the :ref:`Contribution Guide <screenshots>`.
-
+  ODK Docs contributors can use the :ref:`screenshot utility script <screenshots>`, which wraps the :command:`adb` commands and assists with saving the images to the correct location and inserting appropriate markup in the documentation source.
+  
 .. _recording-video-with-adb:
 
-Recording a video
+Recording video
 ~~~~~~~~~~~~~~~~~~~
 
-:command:`adb` can be used to record video on device's screen. This can be done by running:
+.. code-block:: console
+
+  $ adb shell screenrecord /sdcard/video-name.mp4
+
+When you hit :guilabel:`Enter`, the video starts recording immediately.
+
+To stop the recording, press :kbd:`CTRL-C`. If you don't interrupt the recording, it will stop after three minutes.
+
+To pull the video locally:
 
 .. code-block:: console
 
-  $ adb shell screenrecord /sdcard/example.mp4
+  $ adb pull /sdcard/video-name.png
+
+.. _adb-debug-logs:
+  
+Pulling logs for debugging
```

Comment:
Minor issue, but `pulling` seems like the wrong word. Capture feels better to me.